erpc.yaml

```diff
@@ -1,21 +1,29 @@
 logLevel: warn
+
 database:
   evmJsonRpcCache:
-    driver: memory
-    maxItems: 1000000000
-    # driver: postgresql
-    # postgresql:
-    #  connectionUri: ${DATABASE_URL}
-    #  table: rpc_cache
+    connectors:
+      - id: memory-cache
+        driver: memory
+        memory:
+          maxItems: 1000000000
+    policies:
```

Comment:
@lyoungblood this cache policy will cache all finalized data forever. you can check this docs to add more cache policies if needed: https://docs.erpc.cloud/config/database/evm-json-rpc-cache#cache-policies

Reply:
What if we don't want to cache finalized data forever, but eject oldest/LRU after we reach some threshold of a few GB of RAM? The thing is most of our requests are for the data at tip/head block and we don't care about it 2 seconds later.

Reply:
hmm good question. few points that come to mind and @aramalipoor can correct me if wrong

* I see you already have set the `maxItems` on the memory cache. that should help with ejecting the old data.

* in terms of the policies, I think since you are already calling mostly data at head block, the finalized policy wouldn't interfere too much. but maybe best to cache all possible `finality states` under a very short `ttl` like `2s`?